modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/checkpoint/CheckpointWorkflow.java

```diff
@@ -63,14 +61,8 @@
  * <p>{@link CheckpointWorkflow#markCheckpointEnd} - Finalization of last checkpoint.
  */
 class CheckpointWorkflow {
-    /**
-     * Starting from this number of dirty pages in checkpoint, array will be sorted with {@link Arrays#parallelSort(Comparable[])} in case
-     * of {@link CheckpointWriteOrder#SEQUENTIAL}.
-     */
-    private final int parallelSortThreshold;
-
-    /** This number of threads will be created and used for parallel sorting. */
-    private static final int PARALLEL_SORT_THREADS = Math.min(Runtime.getRuntime().availableProcessors(), 8);
+    /** Starting from this number of dirty pages in checkpoint, array will be sorted with {@link Arrays#parallelSort(Comparable[])}. */
+    static final int PARALLEL_SORT_THRESHOLD = 40_000;
```

Comment:
The link is in the ticket, do you think there is a need to indicate in the comments?

Reply:
Imagine that after a year or two there are 50 commits in the history of this file, and the line we are discussing is changed a few times (maybe the field is made public, the line moved, then the field is made package local, etc). It may become really difficult to find the commit which introduced this constant, and we need to find the commit to find the ticket. But if we put the link to a comment, there is hope that the comment will not get mutilated or removed.

Reply:
fix it